File: benchmarks/fib42/fib42.cs
using System;

class Program {
    static long fib(int n) {
        if (n <= 1) return n;
        return fib(n - 1) + fib(n - 2);
    }

    static void Main() {
        int n = 42;
        long result = fib(n);
        Console.WriteLine(result);
    }
}

File: benchmarks/fib42/fib42.cpp
#include <iostream>

long long fib(int n) {
    if (n <= 1) return n;
    return fib(n - 1) + fib(n - 2);
}

int main() {
    int n = 42;
    long long result = fib(n);
    std::cout << result << std::endl;
    return 0;
}
